charts/scalar-manager/values.yaml

```
@@ -80,12 +80,69 @@ api:
     paused-state-retention.storage=${PAUSED_STATE_RETENTION_STORAGE:configmap}
     paused-state-retention.max-number=${PAUSED_STATE_RETENTION_MAX_NUMBER:100}
 
+    # OpenAPI configuration
+    springdoc.swagger-ui.enabled=${SPRINGDOC_SWAGGER_UI_ENABLED:false}
+    springdoc.swagger-ui.path=${SPRINGDOC_SWAGGER_UI_PATH:/swagger-ui.html}
+
+    # CORS configuration
+    app.cors.allowed-origins=${APP_CORS_ALLOWED_ORIGINS:*}
+    app.cors.allowed-methods=${APP_CORS_ALLOWED_METHODS:*}
+    app.cors.allowed-headers=${APP_CORS_ALLOWED_HEADERS:*}
+
+    # Whether to enable persistence endpoints or not (auth, user management)
+    app.persistence-endpoints.enabled=${APP_PERSISTENCE_ENDPOINTS_ENABLED:false}
+
+    # Whether to enable authorization or not for the operational endpoints
+    app.authorization.enabled=${APP_AUTHORIZATION_ENABLED:false}
+
+    # JWT configuration
+    # P-256 (secp256k1) private key in PKCS8 format, using for signing JWT tokens, required when persistence endpoints are enabled
+    authentication.providers.static-jwt.private-key=${AUTHENTICATION_PROVIDERS_STATIC_JWT_PRIVATE_KEY:}
+    # Public key in X.509/SPKI format using for verifying JWT tokens, when authentication is enabled, this key is used for verifying JWT tokens
+    authentication.providers.static-jwt.public-key=${AUTHENTICATION_PROVIDERS_STATIC_JWT_PUBLIC_KEY:}
+    authentication.providers.static-jwt.issuer-uri=${AUTHENTICATION_PROVIDERS_STATIC_JWT_ISSUER_URI:https://scalar-manager.scalar-labs.com}
```

```suggestion
    authentication.providers.static-jwt.issuer-uri=${AUTHENTICATION_PROVIDERS_STATIC_JWT_ISSUER_URI:https://scalar-manager.example.com}
```

Is this URL `https://scalar-manager.scalar-labs.com` provide some actual service? Or, is this just a sample?

If it's a sample, I think it might be better to use another domain like `example.com` that is described in [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.html).